python/test/t_oscillator.py

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env python
+
+from openturns.testing import assert_almost_equal
+from openturns.usecases import oscillator
+
+"""
+Test the WingWeightModel data class on a reference point.
+"""
+osc = oscillator.Oscillator()
+
+
+assert_almost_equal(osc.model(osc.distribution.getMean()), [23.1897], 1e-2)
```

Comment:
This unit test is almost empty. And, indeed, testing such a difficult benchmark function is not that easy (otherwise we would not create it). I feel, however, that any bug in the code may become unnoticed.

Please consider using a more thorough testing of the attributes, such as the fact that the most advanced FORM-IS method provides a rough approximation of the reference probability without being too costly (say, within 1 second or so). This will make maintenance much more easy. You may use the unit tests from otbenchmark as a reference : https://github.com/mbaudin47/otbenchmark/blob/master/tests/test_RelabilityProblem14.py

Reply:
Thanks for your remark. I added a test using FORM.